test/models/order_test.rb

```diff
@@ -31,8 +31,8 @@ def setup
     assert_equal 'John', user_first_name
   end
 
-  test "order has items associated with it" do
-    item = order.items.create(title: 'coffee', description: "akjdf", price: 203921)
-    assert_equal 'coffee', order.items.first.title
+  test "order has projects associated with it" do
+    project = order.projects.create(title: 'coffee', description: "akjdf", price: 203921)
```

Comment:
Useless assignment to variable - `project`.<br>Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.